src/vorta/utils.py

```diff
@@ -191,35 +196,21 @@ def get_private_keys():
             key_file = os.path.join(ssh_folder, key)
             if not os.path.isfile(key_file):
                 continue
-            for key_format in key_formats:
-                try:
-                    parsed_key = key_format.from_private_key_file(key_file)
-                    key_details = {
-                        'filename': key,
-                        'format': parsed_key.get_name(),
-                        'bits': parsed_key.get_bits(),
-                        'fingerprint': parsed_key.get_fingerprint().hex(),
-                    }
-                    available_private_keys.append(key_details)
-                except (
-                    SSHException,
-                    UnicodeDecodeError,
-                    IsADirectoryError,
-                    IndexError,
-                    ValueError,
-                    PermissionError,
-                    NotImplementedError,
-                ):
-                    logger.debug(
-                        f'Expected error parsing file in .ssh: {key} (You can safely ignore this)', exc_info=True
-                    )
+            # ignore config, known_hosts*, *.pub, etc.
+            if key.endswith('.pub') or key.startswith('known_hosts') or key == 'config':
+                continue
+            try:
+                if is_ssh_file(key):
+                    available_private_keys.append(key)
+            except (PermissionError,):
```

Comment:
@m3nu in which cases is a permission error expected?

Reply:
It's possible. Like when people run commands as root, while `$HOME` is still set for their own user.

Reply:
We should also add the `UnicodeDecodeError`. Else it crashes for me while parsing a macOS-specific file in that folder

```
2023-02-24 12:58:35,597 - root - DEBUG - Expected error parsing file in .ssh: .DS_Store (You can safely ignore this)
Traceback (most recent call last):
  File "/Users/manu/Workspace/vorta/src/vorta/utils.py", line 201, in get_private_keys
    if is_ssh_file(key_file):
       ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/manu/Workspace/vorta/src/vorta/utils.py", line 181, in is_ssh_file
    first_line = f.readline()
                 ^^^^^^^^^^^^
  File "<frozen codecs>", line 322, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 3131: invalid start byte
```